service/src/main/java/com/commercetools/pspadapter/payone/transaction/TransactionBaseExecutor.java

```diff
@@ -55,4 +59,36 @@ protected SetStatusInterfaceText setStatusInterfaceText(final Map<String, String
         }
         return SetStatusInterfaceText.of(stringBuilder.toString());
     }
+
+    /**
+     * Converts {@code Map<String, String>} response to a string with a JSON value.
+     * @param response Response map to convert to a JSON String
+     * @return JSON string with respective {@code response} key-value entries.
+     * @see #exceptionToResponseJsonString(Exception)
+     */
+    protected static String responseToJsonString(Map<String, String> response) {
+        return SphereJsonUtils.toJsonString(response);
+    }
+
+    /**
+     * In case an error occurred on commercetools service side, but not in Payone, we simulate same response structure,
```

Comment:
it is not "commercetools service side" but for example connection error on connection to payone servers (timeouts, authorisation...)